src/rss.11ty.js

```diff
@@ -0,0 +1,71 @@
+const absoluteUrl = require('@11ty/eleventy-plugin-rss/src/absoluteUrl');
+const dateToISO = require('@11ty/eleventy-plugin-rss/src/dateToISO');
+
+const xml = String.raw;
+
+exports.data = () => {
+  return {
+    permalink: 'feed.xml',
+    site: {
+      title: 'Oliver Phillips - Frontend Engineer',
+      subtitle: 'I design and develop user experiences.',
+      url: 'https://oliverjam.es/',
+      author: {
+        name: 'Oliver Phillips',
+        email: 'hello@oliverjam.es'
+      },
```

Comment:
I should probably move this stuff into a global data file so I can edit it in one place. Not a blocker for this PR though, I'll get round to it at some point